### DISCUSSION TITLE
Create a new plugin for compressing files

### DISCUSSION THREAD

**Discussion by TheStxfan** [Flow-Launcher/Flow.Launcher#2368] [Ideas]
Recently I used the 7-zip.exe and 7-zip.dll to create some batch files that compress to .7z any type of files drag and dropped onto them. (folders and files). 
They are 6 files which compress at different compress ratio. From very low (faster but low compress ratio) to Extreme (slower but high compression ratio).
It'll be nice if I see them integrated into flow launcher. I suggest the very low one, as it compresses a lot but at faster speed.
I am new to coding so I don't know how to integrate it into flow launcher by myself but I hope I to become a contributor anyway if you decide to integrate my project.
And thanks for Flow Launcher, I love it!
This is the project: https://github.com/TheStxfan/BatchUltraFileCompression

**Comment by jjw24** [comment]
Not sure why I can't change this Category to plugin request so going to try convert it to issue and then back.

**Reply by jjw24** [reply]
Transferred: https://github.com/Flow-Launcher/Flow.Launcher/discussions/2374

**Reply by TheStxfan** [reply]
Thank you! :)
